character/templates/character/home.html

```diff
@@ -17,22 +17,22 @@ <h1>List of Users</h1>
 <table id="userList" class="table table-striped table-bordered table-sm" cellspacing="0" width="100%">
   <thead>
     <tr>
-      
+
       <th scope="col" onclick="sortTable(0)">User</th>
       <th scope="col" onclick="sortTable(1)">Corporation</th>
-     
+
     </tr>
   </thead>
   <tbody>
     <tr>
-      
+
       {% for character in character_list %}
```

Comment:
This needs to move one line up, otherwise you do not generate a new TR for each character in the list and that does not produce a clean html code although browser kind of fix it for you.